.storybook/stories/DataFiltering/components/DashboardTable.tsx

```diff
@@ -197,11 +197,14 @@ export const DashboardTable = forwardRef<HTMLTableElement, DashboardTableProps>(
 												<TableCell>
 													{city}, {state}
 												</TableCell>
-												<TableCell textAlign="right">
```

Comment:
After chatting with @danger17 , we realised that this is incorrect. Only numbers should be right aligned, not dates.